smac/facade/smac_facade.py

```diff
@@ -32,7 +33,9 @@ class SMAC(object):
 
     def __init__(self,
                  scenario: Scenario,
-                 tae_runner: ExecuteTARun=None,
+                 # TODO: once we drop python3.4 add type hint
```

Comment:
Typing is not available in python3.4, so I can't do it right away. You're sure I should not leave it in there?